test-cases/longevity/longevity-large-partition-8h.yaml

```diff
@@ -34,8 +34,8 @@ post_prepare_cql_cmds: "CREATE MATERIALIZED VIEW scylla_bench.view_test AS SELEC
 #                  "scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=1000 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -rows-per-request=10 -consistency-level=quorum -timeout=30s -retry-number=30 -retry-interval=80ms,1s -partition-offset=1001 -concurrency=100 -connection-count=100 -duration=6420m"
 #                 ]
 
-n_db_nodes: 5
-n_loaders: 4
+n_db_nodes: 3
+n_loaders: 2
```

Comment:
was it changed in purpose, or just for testing this PR?

Reply:
return